frontend/src/components/Board/Board.js

```diff
@@ -67,26 +66,54 @@ const Board = () => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [boardArray]);
 
-  const checkBoardCorrect = () => {
-    // Send this board to server
-    // const boardForServer = parseBoard(rows);
-    // Response from server
-    const boardCorrect = true;
-    if (boardCorrect) {
-      downloadNewBoard();
-    }
+  useEffect(() => {
```

Comment:
You should make this effect return a function that cleans up the opened web socket.